app/models/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order < ActiveRecord::Base
+  has_many :order_items
+  belongs_to :user
+
+  include ActiveModel::Validations
```

Comment:
Because this inherits from `ActiveRecord::Base`, you likely don't need this `include`. Please correct me if I'm wrong.